.github/workflows/update-interactive-templates.yml

```
@@ -0,0 +1,54 @@
+---
+name: "Create PR to update interactive-templates"
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "25 13 * * *"
+
+jobs:
+  create_job_server_pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: opensafely-core/setup-action@v1
+        with:
+          install-just: true
+          python-version: "3.12"
+
+      - name: Update requirement to latest
+        run: |
+          just update-interactive-templates
+          updated_version="$(sed -n 's|^interactive_templates@https://github.com/opensafely-core/interactive-templates/archive/refs/tags/\(v[0-9.].*\).zip$|\1|p' requirements.prod.in)"
+          echo "UPDATED_VERSION=$updated_version" >> "$GITHUB_ENV"
+
+      - name: Create a Pull Request if there are any changes
+        id: create_pr
+        uses: peter-evans/create-pull-request@c5a7806660adbe173f04e3e038b0ccdcd758773c # v6.1.0
+        with:
+          token: "${{ secrets.JOB_SERVER_PR_TOKEN }}"
+          add-paths: requirements.*
+          base: main
+          branch: bot/update-interactive-templates
+          author: "opensafely-github-bot <opensafely-github-bot@users.noreply.github.com>"
+          committer: "opensafely-github-bot <opensafely-github-bot@users.noreply.github.com>"
+          commit-message: "feat: Update interactive templates to ${{ env.UPDATED_VERSION }}"
```

We don't need the "feat" prefix, do we? job-server doesn't use conventional commits. Also, how would you feel about making the commit messages consistent with Dependabot's commit messages?

```suggestion
          commit-message: "Bump interactive-templates to ${{ env.UPDATED_VERSION }}"
```